examples/tests/helpers/multisend.ts

```diff
@@ -0,0 +1,41 @@
+import { PersistenceClient, cosmos } from "persistenceonejs";
+
+export async function MultiSend(client: PersistenceClient, input: [], output: []) {
```

Comment:
```suggestion
import {Input ,Output} from "cosmjs-types/cosmos/bank/v1beta1/bank";

export async function MultiSend(client: PersistenceClient, input: Input[], output: Output[]) {
```

since it is TypeScript, we can use imports like these, instead of commenting example of inputs.